<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edge_detection_widget</class>
 <widget class="QWidget" name="edge_detection_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="camera_radiobutton">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>CAMERA</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="systemfile_radiobutton">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>SYSTEM FILE</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="select_file_button">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>SELECT FILE</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="edge_detection_algorithm_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>EDGE DETECTION ALGORITHM</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="edge_detection_algorithm_combobox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="edge_detection_set_colour">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set Colour</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="color_button">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="slider_label_1">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SLIDER 1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QSlider" name="horizontalSlider_1">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="slider_label_2">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SLIDER 2</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="slider_label_3">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SLIDER 3</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="click_image_button">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>CLICK IMAGE</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="restore_button">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>RESTORE</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="save_file_button">
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>SAVE FILE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
